lib/rubocop/cop/rspec/empty_line_after_example_group.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # Checks if there is an empty line after example group blocks.
+      #
+      # @example
+      #   # bad
+      #   RSpec.describe Foo do
+      #   end
+      #   RSpec.describe Bar do
```

Comment:
those seems as offenses for the multipledescribes cop, better add examples for few different describes in a single top-level describe ;)

Reply:
That is a very good point, actually. I’ll change here, and all the specs.